From c2120ccbecf734475fc3a142554a6636fdcd6e5f Mon Sep 17 00:00:00 2001
From: Thomas Sibley <trs@bestpractical.com>
Date: Wed, 3 Apr 2013 12:05:10 -0700
Subject: [PATCH] Add Check/Clear All and Reset buttons for the Quick Search
 (queue list) preferences

Very helpful when the list of queues is long.
---
 share/html/Prefs/Quicksearch.html | 11 ++++++++++-
 1 file changed, 10 insertions(+), 1 deletion(-)

diff --git a/share/html/Prefs/Quicksearch.html b/share/html/Prefs/Quicksearch.html
index 93b026f..cb4292a 100644
--- a/share/html/Prefs/Quicksearch.html
+++ b/share/html/Prefs/Quicksearch.html
@@ -60,7 +60,16 @@ checked="checked"
 /><%$queue->Name%><% $queue->Description ? ': '.$queue->Description : '' %></li>
 % }
 </ul>
-<& /Elements/Submit, Caption => loc("Save Changes"), Label => loc('Save'), Name => 'Save'&>
+
+<& /Elements/Submit,
+    Caption             => loc("Save Changes"),
+    Label               => loc('Save'),
+    Name                => 'Save',
+    Reset               => 1,
+    CheckAll            => 1,
+    ClearAll            => 1,
+    CheckboxNameRegex   => '/^Want-/',
+    &>
 
 </form>
 
-- 
1.7.11.3

